lib/PlacesProvider.js

```diff
@@ -668,6 +669,88 @@ Links.prototype = {
   }),
 
   /**
+   * Gets the remote tabs links.
+   *
+   * @param {Integer} options
+   *
+   *
+   * @returns {Promise} Returns a promise with the array of links as payload.
+   */
+  getRemoteTabsLinks:  Task.async(function*(options = {}) {
+    let {limit} = options;
+    if (!limit || limit.options > LINKS_QUERY_LIMIT) {
+      limit = LINKS_QUERY_LIMIT;
+    }
+
+    // We first get the URLs open on other devices from SyncedTabs.
+    let clients = yield SyncedTabs.getTabClients();
+
+    if (clients.length === 0) {
+      // No reason to waste cycles doing anything else.
+      return;
```

Comment:
lint: should return something although the jsdoc says Promise while below seems to be just an array.